zlibWrapper/examples/example.c

```diff
@@ -62,11 +62,14 @@ void *myalloc(q, n, m)
     unsigned n, m;
 {
     q = Z_NULL;
-    return calloc(n, m);
+    void *buf = calloc(n, m);
```

Comment:
A declaration after a statement.
gcc/clang will run fine, but some compilers will not like it.

Since it's only an example file, it's not a high priority item.
That being said, I recommend the usage of compilation flag `-Wdeclaration-after-statement` to detect such cases early.